celadon_ivi/mixins.spec

```diff
@@ -23,15 +23,15 @@ allow-missing-dependencies: true
 dexpreopt: true
 pstore: ram_dummy(address=0x50000000,size=0x400000,record_size=0x4000,console_size=0x200000,ftrace_size=0x2000,dump_oops=1)
 bugreport: true
-media: mesa(add_sw_msdk=false, opensource_msdk=true, opensource_msdk_omx_il=true)
+media: mesa(enable_msdk_omx=true, add_sw_msdk=false, opensource_msdk=true, opensource_msdk_omx_il=true)
 graphics: mesa(gen9+=true,vulkan=true,minigbm=true,gralloc1=true,enable_guc=false)
 storage: sdcard-mmc0-usb-sd(adoptablesd=true,adoptableusb=true)
 ethernet: dhcp
 camera-ext: ext-camera-only
 rfkill: true(force_disable=)
 wlan: iwlwifi(libwifi-hal=true)
-codecs: configurable(hw_ve_h265=true, hw_vd_vp9=true, hw_vd_mp2=true, hw_vd_vc1=false, platform=bxt,profile_file=media_profiles_1080p.xml)
-codec2: true
+codecs: configurable(sw_omx_video=false, hw_omx_video=true, hw_ve_h265=true, hw_ve_h264=true, hw_vd_vp9=true, hw_vd_mp2=true, hw_vd_vp8=true, hw_vd_h265=true, hw_vd_h264=true, platform=bxt, profile_file=media_profiles_1080p.xml)
+codec2: true(enable_msdk_c2=false, use_onevpl=false)
```

Comment:
codec2 has to be set to true, right? can it set to false.

Reply:
This is just a tag, like configurable in codecs. The variable enable_msdk_c2 is used to  config hw codec2.0. Before without video codec2.0, this tag codec2 is used to enable sw audio and video codec2.0.

Reply:
ok thanks